tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/fsm/EmbeddedClassifier.scala

```diff
@@ -41,22 +46,6 @@ case class EmbeddedClassifier(
       createDTFeatureVector(featureVector)
     )
     dist map { case (transitionIndex, prob) => (transitions(transitionIndex), prob) }
-
-    /*
-    val categoryCounts: Map[Int, Double] = decisionTree.categoryCounts(
-      createDTFeatureVector(featureVector)) mapValues { _.toDouble }
-    val defaultCount: Double = 0.1
-    val smoothedCategoryCounts =
-      (transitions.zipWithIndex map { case (transition, transitionIndex) =>
-        val categoryCount: Double = categoryCounts.getOrElse(transitionIndex, defaultCount)
-        (transition, categoryCount)
-      }).toMap
-    val normalizer: Double = smoothedCategoryCounts.values.sum
-    val result = smoothedCategoryCounts mapValues { case count =>
-      count / normalizer
-    }
-    result
-    */
   }
 
   private def createDTFeatureVector(featureVector: FeatureVector): DTFeatureVector = {
```

Comment:
Same comment as above. I don't know what you're gaining by serializing the classifiers, if you're going to read them out again and put them all into one array.

Unless training itself takes so much memory that you don't want to have the previous classifiers in memory at the same time. Is that it? How big are those classifiers?